Kitodo/src/main/java/org/kitodo/production/services/data/BatchService.java

```diff
@@ -303,39 +305,45 @@ public int size(Batch batch) {
      * batch.
      *
      */
-    public String toString(Batch batch) {
+    public void toString(Batch batch) {
```

Comment:
shouldn't this be named "createLabel" or something?

Reply:
renamed